source/plugins/go/input/go.mod

```diff
@@ -14,14 +14,14 @@ require (
 	github.com/google/uuid v1.6.0 // indirect
 	github.com/ugorji/go/codec v1.2.12 // indirect
 	golang.org/x/mod v0.17.0 // indirect
-	golang.org/x/sync v0.10.0 // indirect
-	golang.org/x/sys v0.28.0 // indirect
+	golang.org/x/sync v0.11.0 // indirect
+	golang.org/x/sys v0.30.0 // indirect
 	golang.org/x/tools v0.21.1-0.20240508182429-e35e4ccd0d2d // indirect
 )
 
 require (
 	Docker-Provider/source/plugins/go/src v0.0.0
-	github.com/golang-jwt/jwt/v5 v5.2.1
+	github.com/golang-jwt/jwt/v5 v5.2.2
```

Comment:
This PR addresses this issue - https://github.com/microsoft/Docker-Provider/issues/1403, right?

Reply:
I will need to update windows as well, let me check